tests/PySys/mqtt_port/mqtt_port_set/run.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import time
+
+from pysys.basetest import BaseTest
+
+
+"""
+Validate changing the mqtt port using the tedge command
+
+Given a configured system
+When mqtt.port is set using tedge mqtt.port set with sudo
+When the tedge config is listed and searched for the port that has been set 
+
+"""
+
+class MqttPortSet(BaseTest):
+    def setup(self):
+        self.tedge = "/usr/bin/tedge"
+        self.sudo = "/usr/bin/sudo"
+
+        self.addCleanupFunction(self.mqtt_cleanup)
+
+    def execute(self):
+        # set a new mqtt port for local communication
+        mqtt_port = self.startProcess(
+            command=self.sudo,
+            arguments=[self.tedge, "config", "set", "mqtt.port", "8880"],
+            stdouterr="mqtt_port_set",
+        )
+
+    def validate(self):
+        tedge_get = self.startProcess(
+            command=self.sudo,
+            arguments=[self.tedge, "config", "list"],
+            stdouterr="tedge_get",
+        )
```

Comment:
I am a little bit concerned that we are using `tedge config` to validate config. Should we not read the file directly for the content rather than use command which we are actually testing?

Reply:
I do not think so. Because tedge config list should list the newly updated configs. This indirectly tests the tedge config list command as well right?

Reply:
Here in this case I would propose to check both sources. But then we should also check, the same way, that the value is set back when calling unset. In other testcases we can then rely on this feature to be working.